src/text.rs

```diff
@@ -91,7 +91,7 @@ pub struct Font {
 }
 
 impl Font {
-    pub(crate) fn new(
+    pub(crate) fn new_owned(
```

Comment:
can we re-use the [generic interface](https://docs.rs/gfx_glyph/0.10.1/gfx_glyph/fn.font.html) of `g::font` here to have a single signature accepting `T: Into<SharedBytes<'a>>`?